src/main/java/school/hei/haapi/service/utils/DataFormatterUtils.java

```diff
@@ -11,6 +13,19 @@
 public class DataFormatterUtils {
   private DataFormatterUtils() {}
 
+  public static String formatLocalDate(LocalDate localDate, String format) {
+    DateTimeFormatter pattern = DateTimeFormatter.ofPattern(format);
+    return localDate.format(pattern);
+  }
+
+  public static String localDateToCommonDate(LocalDate localDate) {
+    DateTimeFormatter formatter =
+        DateTimeFormatter.ofLocalizedDate(FormatStyle.LONG)
+            .withLocale(Locale.FRENCH)
+            .withZone(ZoneId.of("UTC+3"));
+    return formatter.format(localDate);
+  }
```

Comment:
rename to formatLocalDate,

You can have multiple functions with the same name, same return type but different number of parameters